proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,157 +44,127 @@ direction of the policies set by the Board.
-
-  * If the number of Qualified Nominees are equal to or less than the number of TOC seats available to be elected, the Qualified Nominees shall be approved after the nomination period has closed. If there are more Qualified Nominees than open TOC seats available for election by either the the Governing Board, End User TAB or TOC, then that group shall elect the TOC members via a Condorcet vote. A Condorcet vote shall be run using the Condorcet-IRV method through the Cornell online service ([*http://civs.cs.cornell.edu/*](http://civs.cs.cornell.edu/)).
-
-  * If there are fewer Qualified Nominees than open TOC seats available for election by either the Governing Board, End User TAB or TOC, the group shall initiate another round of nominations with each member or individual eligible to nominate submitting at most one (1) nominee.
-
-* Constraints
-
-  *  TOC Members shall serve two-year, staggered terms. The initial six elected TOC members from the Governing Board election shall serve an initial term of three (3) years. The TOC members initially elected by the End User TAB and TOC shall serve an initial term of two (2) years.
-
- * TOC members may be removed by a two-thirds vote of the other TOC members, with the impacted individual ineligible to participate in the vote.
-
- * Any TOC member that misses three (3) consecutive meetings shall be automatically suspended from eligibility to vote until having attended two meetings consecutively. For avoidance of doubt, the suspended TOC member shall be eligible to vote in the second consecutive meeting.
-
- * The TOC charter, model, approach, composition, etc may be modified by a two-thirds vote of the entire Governing Board.
-
-===========
-
+The CPC is an inclusive group with two types of members:
+
+* Regular members
+* Voting members
+
+### Regular members
+
+Regular members have the roles and responsibilities
+outlined below. Regular member is implied when memership type is not
+specified. Anyone who has been a member of one of the projects under the 
+Foundation for at least three months may request to
+become a regular member by opening a PR to add themselves to the list of
+regular members. Regular members remain for as long as they are active
+within the work of the CPC. Regular members who have not been active
+in GitHub, participated in meetings, or other work of the CPC for 3 months may be
+removed from the list of Regular members. In addition, a Regular CPC member
+can be removed by voluntary resignation, or by a standard CPC motion.
+
+### Voting members
+
+Voting members are selected as follows:
+
+* Each top level project may nominate two members through a process
+of their choosing. Once nominated the member must be ratified by the
+board before becoming a Voting member.
+
+* two Voting members may be nominated by the non top-level projects based
+on a process set by the CPC.
+
+* two Voting members may be nominated by the Regular members. Once
```

Comment:
As I don't think we need regular members I think this section should be removed. Even if we kept the regular members this seems excessive

Reply:
In what sense is it excessive.  My thinking is that we will have people who are active in the projects and therefore represent the projects on the CPC.  At the same time we may have people who are very active at the CPC level but less so at the project level.  I think we should have some ability to get those people into the Voting group. 

I w

Reply:
I will update to change ratification to be by the CPC as a first step.